buildozer/def.bzl

```diff
@@ -77,13 +77,12 @@ _buildozer = rule(
             default = True,
         ),
         "_buildozer": attr.label(
-            default = Label("@com_github_bazelbuild_buildtools//buildozer"),
+            default = "@com_github_bazelbuild_buildtools//buildozer",
```

Comment:
What did you learn that helped you make this change?

Reply:
`default` for a `attr.label` attribute type can be a `Label` or a `string` (will be turned into a `Label`). Less text is better =)